Assets/Developers/Lab/Components/Move.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using Unity.Collections;
+using Unity.Entities;
+using Unity.Mathematics;
+using UnityEngine;
+
+namespace JanusVagabond
+{
+  public enum MoveDirection
+  {
+    LEFT = 0,
+    RIGHT = 1
+  }
+
+  namespace Tag
+  {
+    [Serializable]
+    public struct MoveLeft : IComponentData
+    {
+
+    }
+
+    [Serializable]
+    public struct MoveRight : IComponentData
+    {
+
+    }
+  }
+
+  namespace Data
+  {
+
+    [Serializable]
+    public struct Move : IComponentData
+    {
+      public float Speed;
+      public float MaxSpeed;
+      public float MinSpeed;
+      // public MoveDirection Direction;
```

Comment:
Remove commented out code

Reply:
Will need this eventually but I'm not using it yet. Will add a TODO comment here